### ISSUE TITLE
Unexpected exception when argument type is not provided

### ISSUE THREAD

**Issue by matthieucan** [terrencepreilly/darglint#75]
Hello,
I'm experiencing an issue that I managed to pinpoint to this situation.

Let's assume the file `foo.py`:
```python
def foo(bar):
    """Foo.

    Args:
        bar (): A bar.
    """
    print(bar)
```

When I execute
```shell
$ darglint foo.py
```

all is well. However when I use darglint as a flake8 plugin:
```shell
$ flake8 foo.py
foo.py:1:1: DAR000: Unexpected exception in darglint:
```

This doesn't reproduce when I remove the empty parentheses following `bar`.

Any idea what's going on?

**Comment by terrencepreilly** [comment]
Ah, there's probably an exception being raised during argument type analysis.

**Comment by terrencepreilly** [comment]
Resolved in 58b6f1e6395cfb261872b038155ce4c566c753e8.  Fix should be present in v1.2.3

**Comment by matthieucan** [comment]
Thank you!
